apps/editing-toolkit/editing-toolkit-plugin/wpcom-block-editor-nux/src/welcome-tour/tour-steps.js

```diff
@@ -125,7 +147,14 @@ function getTourSteps( localeSlug, referencePositioning ) {
 			referenceElements: referencePositioning && referenceElements[ 4 ],
 			meta: {
 				heading: __( 'More Options', 'full-site-editing' ),
-				description: __( 'Click the settings icon to see even more options.', 'full-site-editing' ),
+				descriptions: {
+					desktop: __( 'Click the settings icon to see even more options.', 'full-site-editing' ),
+					mobile:
+						localeSlug === 'en' ||
+						hasTranslation?.( 'Tap the settings icon to see even more options.' )
```

Comment:
:ok: This change will be queued for retranslation. We'll use the existing translations in the meantime.